chromium_src/components/permissions/android/permission_dialog_delegate.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_PERMISSIONS_ANDROID_PERMISSION_DIALOG_DELEGATE_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_PERMISSIONS_ANDROID_PERMISSION_DIALOG_DELEGATE_H_
+
+#include "brave/components/permissions/permission_lifetime_utils.h"
+
+#define BRAVE_PERMISSION_DIALOG_DELEGATE_H_                                 \
+ private:                                                                   \
+  void ApplyLifetimeToPermissionRequests(JNIEnv* env,                       \
```

Comment:
I thought you were converting this to anonymous?

Reply:
yep, you're right. the second method could also be anonymized.